test/notification/index.js

```diff
@@ -77,6 +77,17 @@ describe("Notification", function() {
 			});
 		});
 
+    describe("when aps properties are given by setters", function() {
```

Comment:
It seems like the indentation of this code block does not match the indentation of the surrounding code.